try:
    a = int(input("hey give me a number: "))
    print(a)
    
except Exception as e:
    print(e)
    
else: 
    print("I am inside else block")